# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gnome-utils22/gfloppy.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gfloppy/src/badblocks.c.orig	Tue Dec  5 17:38:44 2000
+++ ./gfloppy/src/badblocks.c	Sun Mar 25 03:44:56 2001
@@ -41,6 +41,8 @@
 #include <sys/types.h>
 
 #ifdef HAVE_LINUX_FD_H
+#define _LINUX_STRING_H_
+#define _PPC_STRING_H_ /* Is this still needed? At least it doesn't hurt. */
 #include <linux/fd.h>
 #include <linux/fs.h>
 #endif
--- ./gfloppy/src/gfloppy.c.orig	Tue Dec  5 17:38:44 2000
+++ ./gfloppy/src/gfloppy.c	Sun Mar 25 03:32:29 2001
@@ -31,6 +31,7 @@
 #include <linux/major.h>
 
 #ifdef HAVE_LINUX_FD_H
+#define _LINUX_STRING_H_
 #include <linux/fd.h>
 #include <linux/fs.h>
 #endif
